drivers/unix/dir_access_unix.cpp

```diff
@@ -77,7 +77,7 @@ bool DirAccessUnix::file_exists(String p_file) {
 
 	p_file = fix_path(p_file);
 
-	struct stat flags;
+	struct stat flags {};
```

Comment:
I don't know why `clang-format` enforces a space here when it doesn't bother about other types of initializers. I guess it's confused by the `struct` part?